lib/elixir/unicode/unicode.ex

```diff
@@ -1,16 +1,19 @@
-defmodule String.Unicode do
+defmodule String.Hex do
   @moduledoc false
-  def version, do: {7, 0, 0}
 
-  to_binary = fn
-    "" ->
-      nil
-    codepoints ->
-      codepoints = :binary.split(codepoints, " ", [:global])
-      Enum.reduce codepoints, "", fn(codepoint, acc) ->
-        acc <> << String.to_integer(codepoint, 16) :: utf8 >>
-      end
+  def to_binary(""), do: nil
+
+  def to_binary(codepoints) do
```

Comment:
If you define an anonymous function before all modules, it will still be accessible in the module body. So no need for String.Hex, we can continue the anonymous modules route. :)